server/index.js

```diff
@@ -24,6 +24,31 @@ app.get('/api/images/:restaurantId', (req, res) => {
     });
 });
 
+app.post('/api/images', (req, res) => {
+  const data = req.body;
+  db.insert(data)
+    .then(res.status(201).end())
+    .catch(err => console.log('Error occurs in Post'))
+});
+
+app.put('/api/images/:restaurantId', (req, res) => {
+  const data = req.body;
+  const id = req.params.restaurantId;
+  console.log("=============");
+  console.log(data);
+  console.log(id);
```

Comment:
Logging database changges is good practice, but should be kept to 1 line. Something like: `Database: ${id} updated at ${new Date()}`